easybuild/easyconfigs/l/libxml2/libxml2-2.9.4-GCCcore-4.9.3.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxml2'
+version = '2.9.4'
+
+homepage = 'http://xmlsoft.org/'
+description = """Libxml2 is the XML C parser and 
+toolchain developed for the Gnome project
+ (but usable outside of the Gnome platform)."""
+
+toolchain = {'name': 'GCCcore', 'version': '4.9.3'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'http://xmlsoft.org/sources/',
+    'http://xmlsoft.org/sources/old/'
+]
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = 'CC="$CC" CXX="$CXX" --with-pic --without-python --with-zlib=$EBROOTZLIB'
+
+builddependencies = [
+    ('binutils', '2.25'),
+]
+
+dependencies = [
+    ('zlib', '1.2.8'),
+]
+
+sanity_check_paths = {
+    'files': [('lib/libxml2.a', 'lib64/libxml2.a'), ('lib/libxml2.%s' % SHLIB_EXT, 'lib64/libxml2.%s' % SHLIB_EXT)],
+    'dirs': ['bin', 'include/libxml2/libxml'],
+}
+
+modextrapaths = {'CPATH': ['include/libxml2']}
```

Comment:
After a bit of looking around, it seems that this is indeed the right thing to do.

Consistently applied to recent easyconfigs in #3479, and ehhanced the `libxml2` easyblock (used when Python bindings are enabled) as well at https://github.com/hpcugent/easybuild-easyblocks/pull/981